### PR TITLE
Store build date in AssemblyMetadata

### LINKED ISSUE #43: Wrong build date in About dialog
Despite of my expectation, when installed from Microsoft Store, EventLook.exe/dll's last write time seems to be the time when the package was installed on the system. We need to come up with a different method to determine the correct build date. 

image

### PR DESCRIPTION
**Pull request by 0xced**
Also re-use assembly metadata to fill the about box.

Fixes #43

### CODE REVIEW

File: EventLook/EventLook.csproj [hunk 1]
```diff
@@ -1,19 +1,30 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>net6.0-windows10.0.17763.0</TargetFramework>
     <OutputType>WinExe</OutputType>
-    <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
```
Comment (kmaki565) [inline EventLook/EventLook.csproj:5]: Thank you, I didn't know this method. According to [the doc](https://learn.microsoft.com/en-us/dotnet/core/project-sdk/msbuild-props), AssemblyInfo.cs should be deleted after migrating the attributes. But it still compiles if the file is kept, and some attributes in the .cs file are still seen in the assembly (checked by ILSpy). Interesting...

**Review by kmaki565 — COMMENTED** [review]
Thank you for the PR! 
Actually I already committed [my fix](https://github.com/kmaki565/EventLook/commit/c818c0121e780beafa62c5227fd999f08f117485) to the problem in a different branch, but it looks yours is more sophisticated. 
I suggest two changes before merging:
1. Use DateTime.Now instead of DateTime.Today for the embedded build date. 
2. Discard space-only changes.

File: AboutBoxWpf/AboutControlViewModel.cs [hunk 2]
```diff
@@ -158,15 +150,15 @@ public string Version
     }
 
     private string _VersionAppendix = "";
-    public string VersionAppendix
```
[the remark this answers is not in the document]
Comment (0xced) [reply AboutBoxWpf/AboutControlViewModel.cs]: Done (force pushed) in 699ea2c4ffcd45b815cc9c12ea06b7ca64e6fed3.

File: EventLook/View/MainWindow.xaml.cs [hunk 3]
```diff
@@ -33,7 +33,7 @@ public MainWindow()
         // Here we send a message which is caught by the view model.  The message contains a reference
         // to the CollectionViewSource which is instantiated when the view is instantiated (before the view model).
         WeakReferenceMessenger.Default.Send(new ViewCollectionViewSourceMessageToken() { CVS = (CollectionViewSource)(this.Resources["X_CVS"]) });
-
```
[the remark this answers is not in the document]
Comment (0xced) [reply EventLook/View/MainWindow.xaml.cs]: Done (force pushed) in 699ea2c4ffcd45b815cc9c12ea06b7ca64e6fed3.

File: EventLook/EventLook.csproj (hunk 1 above)
Comment (0xced) [reply EventLook/EventLook.csproj:5]: I have _finished_ the migration in c9892c58041af2e46fdf32ab9b7d9f076ef627a9 by setting the `ThemeInfo` attribute in the csproj too. (The `SupportedOSPlatform `attribute was already automatically added.)

File: EventLook/EventLook.csproj [hunk 4]
```diff
@@ -1,19 +1,30 @@
...
+  </PropertyGroup>
   <PropertyGroup>
     <ApplicationIcon>Asset\favicon.ico</ApplicationIcon>
   </PropertyGroup>
   <PropertyGroup>
     <ApplicationManifest>app.manifest</ApplicationManifest>
   </PropertyGroup>
+  <ItemGroup>
+    <AssemblyMetadata Include="BuildDate">
+      <Value>$([System.DateTime]::Today.ToString("O"))</Value>
```
[the remark this answers is not in the document]
Comment (kmaki565) [reply EventLook/EventLook.csproj:33]: Yes, I know. Just in case we want to show build time info in the UI in the future.

**Pull request merged**